### PR TITLE
jQ

### PR DESCRIPTION
**Pull request by SteepZero388**
added hello & jQ

### CODE REVIEW

**Comment by steepzero** [comment]
Let's discuss the changes

**Comment by steepzero** [comment]
hey, @SteepZero388 What's up?

**Comment by SteepZero388** [comment]
Yap

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 test repo
 
 Hi Humans!
-Hubot here, I like Node.js and Coffeescript
```
Comment (steepzero) [inline README.md]: WTF?

Comment (SteepZero388) [inline README.md]: What?

**Comment by steepzero** [comment]
@SteepZero388 this does not work

**Pull request merged**